guides/common/attributes-base.adoc

```diff
@@ -29,6 +29,9 @@
 // Do not update the puppet4 repo versions. They must stay at 6.3.
 :RepoRHEL7ServerSatelliteServerPuppetVersion: rhel-7-server-satellite-6.3-puppet4-rpms
 :RepoRHEL7ServerSatelliteCapsulePuppetVersion: rhel-7-server-satellite-capsule-6.3-puppet4-rpms
+:RepoRHEL8ServerBaseos: rhel-8-for-x86_64-baseos-rpms
```

Comment:
There is no RHEL 8 Server (nor a RHEL 8 Workstation), there is just RHEL 8. The name should reflect that. In https://github.com/theforeman/foreman-documentation/pull/1054 there was some work to clean it up. That's probably a much better base to work on.